TheJumpCloud/jumpcloud-ADMU
# Sa 3000 admu reverse

## Issues
* [SA-3000](https://jumpcloud.atlassian.net/browse/SA-3000) - SA-3000-ADMU-Reverse

## What does this solve?
Admins will be able to reverse an ADMU migration with the form and through commands.
## Is there anything particularly tricky?
N/A
## How should this be tested?
Normal Migrated User (Non-UpdatedHomepath)

1. Select an already migrated user on the form, this should enable Reverse Migrate checkbox
2. Reverse migrate
3. Validate that the .dat file, registry, and login has been updated

Migrated User with Updated-Homepath

1. Select a migrated user that has it's homepath updated from Start-Migration
2. Reverse migrate
3. Validate that the .dat file, registry, and login has been updated
4. Validate the user folder name C:\Users\YOURUSER is updated to the backup previous ProfileImagePath



[SA-3000]: https://jumpcloud.atlassian.net/browse/SA-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
